### DISCUSSION TITLE
Displaying missing maps when importing collections

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#13194] [Ideas]
**Describe the new feature:** 
If I import a collection from a stable, then the entire list of maps from the collection is shown, but those that are missing from the lazer are offered to be downloaded (in the list of maps, they can be grayed out or you can make a separate section for them where you can download them).

**Comment by frenzibyte** [comment]
I guess it has to be done for the ["import exported collection"](https://github.com/ppy/osu/issues/4651) case. But until then, I see no reason for having it this way, why don't you just import all of your beatmaps from stable, before importing your collections?

**Comment by Moltanicaa** [comment]
because I can have 20,000 maps, which together with the lazer will take up 2 times more space (the maximum osu weight was 200 gigabytes, so multiply by 2).

UPD. this can be useful for lazer on phones

**Comment by frenzibyte** [comment]
If anything, making the missing beatmaps import from stable during collections import may be possible and fine, but would require some effort.
